projects/storefrontapp-e2e-cypress/cypress/helpers/cart-import-export.ts

```diff
@@ -209,19 +209,23 @@ export function verifyImportedData(config: ImportConfig, cart) {
  * @param productCode identifies the unique product to add.
  */
 export function addProductToCart(productCode: string = cart.products[1].code) {
-  cy.intercept('GET', `**/users/*/carts/*?fields=**`).as('refresh_cart');
+  cy.intercept('GET', `**/products/${productCode}**`).as('productPage')
+  cy.intercept('GET', `**/users/*/carts/*?fields=**`).as('refreshCart');
   cy.intercept('POST', `**/users/*/carts/*/entries?**`).as('addToCart');
   cy.visit(`/product/${productCode}`);
+  cy.wait('@productPage');
   cart.clickAddToCart();
-  cy.wait(['@refresh_cart', '@addToCart']);
+  cy.wait(['@refreshCart', '@addToCart']);
 }
 
 /**
  * Goes to the cart page and clicks the button to export cart.
  * @param expectedData will compare the data of the downloaded .csv to the parsed string.
  */
 export function exportCart(expectedData?: string) {
+  cy.intercept('GET', `**ContentPage&pageLabelOrId=%2Fcart**`).as('cart');
```

Comment:
Unsure if this is the best string to use for the intercept

Reply:
same case as above, you can use the generic function to wait for it like 

```
  const cartPage = waitForPage('/cart', 'getCartPage');
  cy.visit('/cart');
  cy.wait(`@${cartPage}`).its('response.statusCode').should('eq', 200);
```

P.S don't forget to have the visit with '/' to make sure it doesn't just relatively add it without putting it as a new path in the url.